build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         publishVersion = '1.13.0'
 
         //dependencies version
-        kotlinVersion = '1.2.40'
+        kotlinVersion = '1.2.50'
```

Comment:
This might be super nitpicking but do you think it is possible to have another PR to update dependencies outside this one? And have this PR rebase to master after that "another" PR is merged? Thanks!

Reply:
done

I have posted some more ideas/questions in the fuel slack channel I would love to hear community feed back